dev-utils/src/commands/genesis_command.rs

```diff
@@ -16,13 +16,13 @@ use chain_core::tx::fee::{LinearFee, Milli};
 use chrono::offset::Utc;
 use chrono::DateTime;
 use kvdb_memorydb::create;
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 use std::collections::BTreeMap;
 use std::fs;
 use std::path::PathBuf;
 use std::sync::Arc;
 
-#[derive(Deserialize)]
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
why does one need to serialise these?
previously it was this dev config one would edit by hand, so this code just needed to deserialise it for processing

Reply:
Oh, I see it's used here: https://github.com/crypto-com/chain/pull/238/files#diff-11498f5b59098eafc02087e776f6782cR77

Why not just construct this struct directly?

Reply:
fixed